File: android (kotlin)/app/src/main/kotlin/jp/cordea/conviviumcalculator/ListItem.kt
package jp.cordea.conviviumcalculator

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey

open class ListItem : RealmObject() {

    @PrimaryKey
    open var name: String? = null

    open var price: Int = 0

    open var isChecked: Boolean = false
}

File: ios (swift)/ConviviumCalculator/ListItem.swift
//
//  ListItem.swift
//  ConviviumCalculator
//
//  Created by Yoshihiro Tanaka on 2016/01/04.
//
//

import Foundation
import RealmSwift
import Realm

class ListItem: Object {
    @objc dynamic var name: String? = nil
    @objc dynamic var price: Int = 0
    @objc dynamic var isChecked: Bool = false
    
    override static func primaryKey() -> String? {
        return "name"
    }
}
